main/views.py

```diff
@@ -23,3 +27,7 @@ def about(request):
 def logout_view(request):
     logout(request)
     return index(request)
+
+
+def profile(request):
+    return daycare_home(request)
```

Comment:
If that's the homepage anyway, is there a real need for a profile page? I think it might be confusing (especially in the navbar), unless we have an additional basic homepage for the website (which I recommend...)

Reply:
We have a single basic homepage that is shared by both daycare and dog-owner users, so the purpose of the profile page for the daycare as we designed is to update the personal profile details of the daycare user, and this is a future page that we have not implemented now and decided it is not our top priority.

Reply:
If you did not implement it, do not create a route for it.

Reply:
Agree
removed the profile implication